test/integration/narrow/inbound.test.js

```diff
@@ -115,6 +138,11 @@ describe('inbound batch event', () => {
       category: BATCH
     }))
   })
+
+  test('does not send alert for batch', async () => {
```

Comment:
Again this test name doesn't seem clear to me

Reply:
Renamed